python/tests/t_shadow.py

```diff
@@ -19,9 +19,7 @@ def init(self) -> None:
         self.options = None
 
     def manifests(self) -> str:
-        s = super().manifests() or ""
-
-        return s + """
+        return super().manifests() + """
 ---
```

Comment:
NB

Reply:
Why did you not swap the order here? Do the tests fail if so?

Reply:
Um.  Good question.  Because my Emacs macro didn't catch it properly?

Reply:
:joy:

I'll expect a change then... 🙂